scripts/build/create_commands.py

```diff
@@ -59,7 +61,14 @@ def copy_entry_points(input_data, build_dir):
 
                     LOG.info("Registering sub-command '%s' installed to "
                              "PATH", command_name)
-                shutil.copy2(input_file, package_bin)
+                # Only the CodeChecker top level script should be copied
```

Comment:
Maybe this can be refactored like so:
```py
if file_name.startswith("codechecker-"):
    ...
    shutil.copy2(input_file, package_bin_cmd)
else:
    shutil.copy2(input_file, package_bin)
```